tests/docker.zunit

```diff
@@ -109,6 +109,49 @@
     _fzf_complete_docker 'docker history '
 }
 
+@test 'Testing completion: docker push **' {
+    echo 0 > docker_mock_times
+
+    docker() {
+        docker_mock_times=$(($(cat docker_mock_times) + 1))
+        echo $docker_mock_times > docker_mock_times
+
+        docker_mock_$docker_mock_times $@
+    }
+
+    docker_mock_1() {
+        assert $# equals 3
+        assert $1 same_as 'images'
+        assert $2 same_as '--format'
+        assert $3 same_as 'table {{.Repository}};{{.ID}};{{.Tag}};{{if .CreatedSince}}{{.CreatedSince}}{{else}}N/A{{end}};{{.Size}}'
+
+        echo 'REPOSITORY;IMAGE ID;TAG;CREATED;SIZE'
+        echo 'alpine;e7d92cdc71fe;latest;18 hours ago;5.59MB'
+        echo '<none>;5425b1e460b6;<none>;1 weeks ago;5.59MB'
```

Comment:
`docker push` is not able to push dangling images. Can you remove them out of the completions?